src/execution/join_hashtable.cpp

```diff
@@ -687,6 +697,24 @@ void ScanStructure::NextMarkJoin(DataChunk &keys, DataChunk &input, DataChunk &r
 	finished = true;
 }
 
+static void SetConstantNull(Vector &vec) {
```

Comment:
This function should not be required anymore since my last changes, both `ConstantVector::SetNull` and `SetVectorType` should recurse correctly into structs now.